File: app/src/main/java/com/zhang/mydemo/java/ui/activity/DateUtilsActivity.java
package com.zhang.mydemo.java.ui.activity;

import android.widget.TextView;

import androidx.appcompat.widget.AppCompatTextView;

import com.zhang.mydemo.R;
import com.zhang.mydemo.base.BaseActivity;
import com.zhang.utilslibiary.utils.DateUtil;

public class DateUtilsActivity extends BaseActivity {

    private AppCompatTextView mJavaTv1;
    private AppCompatTextView mJavaTv2;
    private AppCompatTextView mJavaTv3;
    private AppCompatTextView mJavaTv4;

    @Override
    protected int getLayoutId() {
        return R.layout.activity_date_utils;
    }

    @Override
    protected void initView() {
        mJavaTv1 = findViewById(R.id.java_tv_1);
        mJavaTv2 = findViewById(R.id.java_tv_2);
        mJavaTv3 = findViewById(R.id.java_tv_3);
        mJavaTv4 = findViewById(R.id.java_tv_4);
    }

    @Override
    protected void initData() {
        findViewById(R.id.ivPageBack).setOnClickListener(v->{killMyself();});
        TextView tvPageTitle = findViewById(R.id.tvPageTitle);
        tvPageTitle.setText("时间工具类");
    }

    @Override
    protected void setListener() {
        mJavaTv1.setOnClickListener(v -> mJavaTv1.setText("当前时间戳：" + DateUtil.INSTANCE.getCurrentMillis()));
        mJavaTv2.setOnClickListener(v -> mJavaTv2.setText("当前时间格式为(yy-MM-dd hh-mm-ss)：" + DateUtil.INSTANCE.getCurrentTimeYMDHMS()));
        mJavaTv3.setOnClickListener(v -> mJavaTv3.setText("当前时间格式为(yy-MM-dd)：" + DateUtil.INSTANCE.getCurrentTimeYMD()));
        mJavaTv4.setOnClickListener(v -> mJavaTv4.setText("获取现在时间：" + DateUtil.INSTANCE.getStringDate()));
    }

}

File: app/src/main/java/com/zhang/mydemo/ui/activity/DateUtilsActivity.kt
package com.zhang.mydemo.ui.activity

import android.annotation.SuppressLint
import android.os.Bundle
import com.zhang.mydemo.base.activity.BaseActivity
import com.zhang.mydemo.databinding.ActivityDateUtilsBinding
import com.zhang.utilslibiary.utils.DateUtil.getCurrentMillis
import com.zhang.utilslibiary.utils.DateUtil.getCurrentTimeYMD
import com.zhang.utilslibiary.utils.DateUtil.getCurrentTimeYMDHMS
import com.zhang.utilslibiary.utils.DateUtil.getStringDate
import com.zhang.utilslibiary.utils.singleClick
import kotlinx.android.synthetic.main.activity_date_utils.*
import kotlinx.android.synthetic.main.layout_title.*

@SuppressLint("SetTextI18n")
class DateUtilsActivity : BaseActivity<ActivityDateUtilsBinding>() {

    override fun initData() {
        tvPageTitle.text = "时间工具类"
    }

    override fun setListener() {
        mViewBinding.apply {
            tv1.singleClick {
                tv1.text = "当前时间戳：${getCurrentMillis()}"
            }
            tv2.singleClick {
                tv2.text = "当前时间格式为(yy-MM-dd hh-mm-ss)：${getCurrentTimeYMDHMS()}"
            }
            tv3.singleClick {
                tv3.text = "当前时间格式为(yy-MM-dd)：${getCurrentTimeYMD()}"
            }
            tv4.singleClick {
                tv4.text = "获取现在时间：${getStringDate()}"
            }
        }
    }

    override fun initView(savedInstanceState: Bundle?) {

    }
}